packages/travix-ui-kit/tests/unit/stepper/stepper.constructor.spec.js

```diff
@@ -0,0 +1,34 @@
+import Stepper from '../../../components/stepper/stepper';
+
```

Comment:
No reason to test the constructor as well

Reply:
There are only one reason to test - 100% coverage

Reply:
You can achieve this by mounting the component.